### DISCUSSION TITLE
mushr/repos.yaml: realsense-ros no longer has dev branch

### DISCUSSION THREAD

**Discussion by Travis-Hainsworth** [prl-mushr/mushr#95] [Software]
Good morning! Another FYI:

tl;dr - realsense-ros no longer has a development branch, instead the ros1-legacy branch is required

The [Jetson Nano Driver Installation page](https://mushr.io/hardware/nano_install/) calls:
the [mushr_install.sh](https://github.com/prl-mushr/mushr/blob/master/mushr_utils/install_scripts/mushr_install.sh), which calls:
the [mushr_install_stack.sh](https://github.com/prl-mushr/mushr/blob/master/mushr_utils/install_scripts/mushr_install_stack.sh):
line 43 clones [repos.yaml](https://github.com/prl-mushr/mushr/blob/master/repos.yaml):

<img width="597" alt="image" src="https://user-images.githubusercontent.com/33101501/207121271-7f7af108-d966-4c5d-b570-30b39fd1239d.png">

line 20 calls for the development branch of the [realsense-ros package](https://github.com/IntelRealSense/realsense-ros):

<img width="501" alt="image" src="https://user-images.githubusercontent.com/33101501/207121578-27f5a69f-fd77-4979-b0f0-500433b229e2.png">

Realsense-ros no longer has a development branch for ROS1, I believe the ros1-legacy branch should be chosen instead:

<img width="558" alt="image" src="https://user-images.githubusercontent.com/33101501/207122374-d13f8c8d-6113-43c8-8416-92a757525d5c.png">
